### PR TITLE
Redirect works, and created singleCrawl page and controller to displa…

### PR DESCRIPTION
**Pull request by hkp108**
…y list of rounds

### CODE REVIEW

File: www/js/controllers/CrawlSingleController.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+pubcrawlerApp.controller('CrawlSingleController', ['$state','CrawlService', function($state, CrawlService, $location) {
+
+  var self = this;
+
+  CrawlService.getSingleCrawl($state.params.id).then(
+    function(data){
```
Comment (olmesm) [inline www/js/controllers/CrawlSingleController.js]: space

File: www/js/controllers/CrawlsController.js [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
+pubcrawlerApp.controller('CrawlsController', ['$state','CrawlService', function($state, CrawlService, $location) {
 
   var self = this;
 
   self.crawls = [{name: "No pubs yet."}];
 
+  self.paramId = $state.params.id;
+
   CrawlService.getCrawls().then(
     function(data){
```
Comment (olmesm) [inline www/js/controllers/CrawlsController.js]: spcae

File: www/js/controllers/CrawlsController.js [hunk 3]
```diff
@@ -1,17 +1,25 @@
...
       self.crawls = data;
     });
 
   self.createCrawl = function(crawlData) {
-    CrawlService.createCrawl(crawlData);
-    $state.go('app.crawls');
+    CrawlService.createCrawl(crawlData).then(
+      function(crawl){
```
Comment (olmesm) [inline www/js/controllers/CrawlsController.js]: spce

File: www/js/controllers/CrawlsController.js [hunk 4]
```diff
@@ -1,17 +1,25 @@
...
+        console.log('crawl');
+        console.log(crawl.data.message);
+        console.log(crawl.data.message.id);
+        $state.go('app.crawlSingle', {id: crawl.data.message.id});
```
Comment (olmesm) [inline www/js/controllers/CrawlsController.js]: { xxx }

File: www/js/services/CrawlService.js [hunk 5]
```diff
@@ -4,8 +4,9 @@ pubcrawlerApp.service('CrawlService', ['$http', function($http) {
   self.createCrawl = function(crawlData) {
     return $http.post('http://localhost:3000/crawls', crawlData)
     .success(function(data) {
-      console.log(data);
-      return data;
+      console.log('data.message');
```
Comment (olmesm) [inline www/js/services/CrawlService.js]: remove console.log

File: www/js/controllers/CrawlSingleController.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+      console.log(data);
```
Comment (olmesm) [inline www/js/controllers/CrawlSingleController.js]: remove console.log

File: www/js/controllers/CrawlsController.js [hunk 7]
```diff
@@ -1,17 +1,25 @@
...
     });
 
   self.createCrawl = function(crawlData) {
-    CrawlService.createCrawl(crawlData);
-    $state.go('app.crawls');
+    CrawlService.createCrawl(crawlData).then(
+      function(crawl){
+        console.log('crawl');
+        console.log(crawl.data.message);
+        console.log(crawl.data.message.id);
```
Comment (olmesm) [inline www/js/controllers/CrawlsController.js]: remove console.log

File: www/js/controllers/CrawlsController.js [hunk 8]
```diff
@@ -1,17 +1,25 @@
-pubcrawlerApp.controller('CrawlsController', ['$state','CrawlService', function($state, CrawlService) {
+pubcrawlerApp.controller('CrawlsController', ['$state','CrawlService', function($state, CrawlService, $location) {
 
   var self = this;
 
   self.crawls = [{name: "No pubs yet."}];
 
+  self.paramId = $state.params.id;
```
Comment (hawksdoves) [inline www/js/controllers/CrawlsController.js]: This has been removed in branch 31.
Pull request for branch 31, will resolve these issues. Will after this.

**Comment by olmesm** [comment]
:beer:

**Pull request merged**